README.md

```diff
@@ -144,6 +145,11 @@ When executed with the `--with-description` option, output with short descriptio
  pytz    2017.3   MIT      World timezone definitions, modern and historical
 ```
 
+### Option: with-license-file
+
+When executed with the `--with-license-file` option, output the location of the package's license file on disk and the full contents of that file. Due to the length of these fields, this option is best paired with `--format-json`.
```

Comment:
The document is valid and I agree.

After merging this P-R, I make a warning to recommend the JSON format when running the `--with-license-file` option with table format.

 How do you like it?

Reply:
I agree that a warning to use the JSON format with `--with-license-file` is a good idea.